lib/theme.ts

```diff
@@ -0,0 +1,70 @@
+export const theme = {
+  root: {
+    primaryColor: "rgb(8 117 225)",
+    dangerColor: "salmon",
+    warningColor: "yellow",
+  },
+  sidebar: {
+    logo: "https://images.ctfassets.net/hjneo4qi4goj/gL6Blz3kTPdZXWknuIDVx/7bb7c73d93b111ed542d2ed426b42fd5/flatfile.svg",
```

Comment:
Is there a white text version of this logo? The black is kind of hard to read on the blue bg

Reply:
I dont think I can add the white logo directly in app from the app side. I think I can set it on the config side. 

Im questioning if we should be making changes for this on the config side. It would be awkward though to have spaceProps on the app side that makes changes as well as changing the default on the config side. 

Maybe set the new default theme on the config side and make the subtle changes on the app side?

Reply:
Also, to change the theme for project-onboarding, we need to edit the theme from the config side. Creating a space does not currently allow a space config currently. It only allows a spaceConfigId